resources/shaders/Pbr.frag

```diff
@@ -37,7 +37,6 @@ struct ObjectData
     // N MVP matrices
     mat4 model;
     mat4 modelView;
```

Comment:
I love seeing less data here! 
You wouldn't know this, but we also don't need modelView here. I'll create an issue to remove it as well.